dockerfiles/coturn.Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM coturn/coturn:alpine AS dev
+USER root:root
+RUN apk add curl netcat-openbsd
+COPY files/coturn/certs/key.pem /certs/key.pem
+COPY files/coturn/certs/cert.pem /certs/cert.pem
+COPY files/coturn/entrypoint.sh /entrypoint.sh
+RUN chmod +x /entrypoint.sh
```

Comment:
If you chmod the file in the repo, then you don’t have to in the dockerfile.

Reply:
Can you please add a newline to the end of the file?

Reply:
Are permissions kepts accross different OSs?

Reply:
Affirmative. The file permission comes from the build context.